satchless/checkout/tests/__init__.py

```diff
@@ -77,7 +64,6 @@ def checkout(self, *args, **kwargs):
 
 class App(BaseCheckoutAppTests):
```

Comment:
This no longer has any tests, is it used as a base class for other tests outside of this module? Actually this whole `checkout.tests` module doesn't define any tests that I can see, is it still required?